website/docs/r/pi_volume_attach.html.markdown

```diff
@@ -19,6 +19,11 @@ resource "ibm_pi_volume_attach" "testacc_volume_attach"{
 	pi_volume_id = "<id of the volume to attach>"
 	pi_instance_id = "<pvm instance id>"
 }
+resource "ibm_pi_volume_detach" "testacc_volume_detach"{
+	pi_cloud_instance_id = "<value of the cloud_instance_id>"
+	pi_volume_id = "<id of the volume to detach>"
+	pi_instance_id = "<pvm instance id>"
+}
```

Comment:
There is no ibm_pi_volume_detach resource. You use the ibm_pi_volume_attach resource to do both.